resources/views/auth/application/questions.blade.php

```diff
@@ -48,15 +48,16 @@
                         <p>
                             @php $checked = old('status') ?  old('status') == 'resident' : $user->application->applied_for_resident_status @endphp
                             <label class="black-text">
-                                <input type="radio" name="status" value="resident"
+                                <input type="radio" name="status" value="resident" required
                                     {{ $checked ? 'checked' : '' }}>
                                 <span>@lang('role.resident')</span>
                             </label>
                         </p>
                         <p>
-                            @php $checked = old('status') ?  old('status') == 'extern' : !$user->application->applied_for_resident_status @endphp
+                            @php $checked = old('status') ?  old('status') == 'extern'
+                                    : (!is_null($user->application->applied_for_resident_status) && !$user->application->applied_for_resident_status) @endphp
```

Comment:
I _understand_ what this code is doing, but I refuse to believe that handling a three-state flag is this convoluted in PHP.

Reply:
I've looked it up now; it seems `===` does the job.